src/atmos_param/socrates/interface/socrates_interface.F90

```diff
@@ -55,10 +55,13 @@ MODULE socrates_interface_mod
 !   INTEGER :: id_soc_surf_spectrum_sw !not implemented yet
   INTEGER :: id_soc_tdt_sw, id_soc_tdt_lw, id_soc_tdt_rad
   INTEGER :: id_soc_surf_flux_lw, id_soc_surf_flux_sw 
+
   INTEGER :: id_soc_surf_flux_lw_down, id_soc_surf_flux_sw_down 
+  INTEGER :: id_soc_surf_flux_lw_clear, id_soc_surf_flux_sw_clear 
+  INTEGER :: id_soc_surf_flux_lw_down_clear, id_soc_surf_flux_sw_down_clear 
   INTEGER :: id_soc_flux_lw, id_soc_flux_sw
-  INTEGER :: id_soc_olr, id_soc_toa_sw
-  INTEGER :: id_soc_toa_sw_down
+  INTEGER :: id_soc_olr, id_soc_toa_sw, id_soc_olr_clear, id_soc_toa_sw_clear
+  INTEGER :: id_soc_toa_sw_down, id_soc_toa_sw_down_clear
```

Comment:
Hi @penmaher, thank you very much for adding more outputs about the clear-sky fluxes, which makes the calculation of cloud radiative effect much easier.